vroom/perltidy.vroom

```diff
@@ -0,0 +1,38 @@
+Perltidy is a linter and formatter for perl.
+If you aren't familiar with basic codefmt usage yet, see main.vroom first.
+
+First, set up the vroom environment.
+
+  :source $VROOMDIR/setupvroom.vim
+
+  :let g:repeat_calls = []
+  :function FakeRepeat(...)<CR>
+  |  call add(g:repeat_calls, a:000)<CR>
+  :endfunction
+  :call maktaba#test#Override('repeat#set', 'FakeRepeat')
+
+  :call codefmt#SetWhetherToPerformIsAvailableChecksForTesting(0)
+
+By default, the perltidy executable is called. To use this plugin, perltidy
+must be installed on your system. (But not for testing; vroom intercepts
+system calls.)
+  :FormatCode perltidy
+  ! perltidy .*
```

Comment:
You need to fake the command output like this: https://github.com/google/vim-codefmt/blob/c6730fb9c0d17060ea0caf5b9bd3090e86f6a14d/vroom/clangformat.vroom#L46

Otherwise the tests can't run without perltidy installed (and may get brittle, failing depending on the _version_ of perltidy installed).

Reply:
Updated, and I'm not sure if I'm doing right because I don't really understand the syntax of `.vroom`.